### PR TITLE
Added Alert and Button

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Added Alert And Button

### CODE REVIEW

File: Button/button.css [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    display: none;
+}
+
+.alert-square.show{
+    height:3rem;
+    border: solid red;
+    width: 15rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (sanjay-xdr) [inline Button/button.css:249]: try to create a separate class for display:flex and other properties which you are using many times

File: Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+.center{
+    border: solid;
+    width: 70rem;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-bottom:1rem;
+
+}
```
Comment (sanjay-xdr) [inline Alert/alert.css]: remove these unwanted code

File: Alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    color: #c201018e;
+}
+.center .alert-cont .alert.alert-warning{
+    background-color: #FFF3CD;
+    color: #AC934F;
+}
+.center .alert-cont .alert.alert-dark{
+    background-color: #1f1e1e;
+    color: #eceff1;
+}
```
Comment (sanjay-xdr) [inline Alert/alert.css]: use variables for colors

File: Alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    color: #8D9295;
+}
+.center .alert-cont .alert.alert-primary{
+    background-color: #CCE5FF;
+    color: #5882AE;
+}
+    
+.code-box {
+    /* max-width: 75vw; */
+    background: #eee;
```
Comment (devananth) [inline Alert/alert.css:72]: Remove commented lines.

File: Alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+    color: #8AA98D;
+}
+.center .alert-cont .alert.alert-light{
+    background-color: #E2E3E5;
+    color: #8D9295;
+}
+.center .alert-cont .alert.alert-primary{
+    background-color: #CCE5FF;
+    color: #5882AE;
+}
```
Comment (devananth) [inline Alert/alert.css]: Use `css variables` for colors

**Review by devananth — CHANGES_REQUESTED** [review]
Good Work ! 

Few Suggestions : 
      - Remove the commented lines before raising the PR
      - Use `css variables` for colors
      - Add `image` or `GIF` or `preview link` , unable to visualise how the page would look like

File: Alert/alert.css [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+    padding: 1rem;
+    width: 50%;
+    color: black;
+    font-size: 500;
+    border-radius: 0.5rem;
+    border: solid;
+}
+.center .alert-cont .alert.alert-danger{
+    background-color: var(--danger);
+    color: #c201018e;
```
Comment (md-amir-gauhar) [inline Alert/alert.css:47]: Try using variables for colors. Else looks good👍

**Review by md-amir-gauhar — APPROVED** [review]

File: Alert/alert.css [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+    width: 50%;
+    color: black;
+    font-size: 500;
+    border-radius: 0.5rem;
+}
+.center .alert-cont .alert.alert-simple{
+    padding: 1rem;
+    width: 50%;
+    color: black;
+    font-size: 500;
```
Comment (Hamza-777) [inline Alert/alert.css:41]: font-size or font-weight?

File: Alert/alert.css [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+  /* Links inside the dropdown */
+  .dropdown-content a {
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+  }
+  
+  /* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: #3e8e41;border-radius: 3rem 0rem 0rem 3rem; color: white;margin-top: 0.1rem;text-shadow: 2px 2px 1px black;}
```
Comment (Hamza-777) [inline Alert/alert.css:157]: Format the code properly.

**Review by Hamza-777 — APPROVED** [review]
Everything else looks good.

**Pull request merged**